Description: Fix gray background on dvips in pstoimg
 This patch fixes gray background detection on dvips via a regexp adjustment in
 pstoimg.pin
Author: Diab Jerius <djerius@cfa.harvard.edu>
Bug-Debian: http://bugs.debian.org/291420

--- latex2html-2008-debian1.orig/pstoimg.pin
+++ latex2html-2008-debian1/pstoimg.pin
@@ -1137,7 +1137,7 @@ sub ps2pnm {
       }
     my $had_nonwhite;
     if($opt{white}) {
-      $had_nonwhite = ($ps =~ s/(\n\d+ \d+ bop gsave) \d*\.\d+ (TeXcolorgray clippath fill grestore)/$1 1 $2/s);
+      $had_nonwhite = ($ps =~ s/(\d+ \d+ bop gsave) \d*\.\d+ (TeXcolorgray clippath fill grestore)/$1 1 $2/s);
       }
     $ps_changed = $had_papersize || $had_nonwhite;
     if($ps_changed) {
